docs/usage.rst

```diff
@@ -97,3 +97,27 @@ Please refer to this section for more details.
 .. _installation: installation.rst
 .. _permissions: permissions.rst
 .. _utilities: utilities.rst
+
+Testing
+~~~~~~~
+
+To execute the tests on your local machine, a PostgreSQL instance must be running.
+This PostgreSQL instance can be easily initialized by using the docker-compose.yml.example file with the default values.
```

Comment:
use references for `PostgreSQL` and `docker-compose.yml.example`
you can check in configuration.rst, they are already defined in there.